server/metrics.go

```diff
@@ -22,8 +22,8 @@ var (
 		prometheus.HistogramOpts{
 			Namespace: "tidb",
 			Subsystem: "server",
-			Name:      "handle_query_duration_seconds",
-			Help:      "Bucketed histogram of processing time (s) of handled queries.",
+			Name:      "handle_query_duration_nanoseconds",
+			Help:      "Bucketed histogram of processing time (ns) of handled queries.",
 			Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 13),
```

Comment:
I prefer Seconds here, like all other programs (etcd, pd) using prometheus.
And here the buckets is 0.0005, it is for seconds, means the lease is 0.5ms, if you use nano, too small.